components/libc/compilers/common/ctime.c

```diff
@@ -626,7 +626,7 @@ RTM_EXPORT(clock_gettime);
 int clock_nanosleep(clockid_t clockid, int flags, const struct timespec *rqtp, struct timespec *rmtp)
 {
     struct timespec ts = {0};
-    rt_err_t        err;
+    rt_err_t err = RT_EOK;
```

Comment:
默认为有错误的错误码，防止走到非正常路径 但是返回正确
```suggestion
    rt_err_t err = -RT_EINVAL;
```